Source/Management.GraphQL/Management.GraphQL.csproj

```
@@ -0,0 +1,17 @@
+<Project Sdk="Microsoft.NET.Sdk">
+    <Import Project="../../default.props"/>
+
+    <PropertyGroup>
+        <AssemblyName>Dolittle.Runtime.Management.GraphQL</AssemblyName>
```

Not sure why this is here - but it is in all the other projects.
```suggestion
        <AssemblyName>Dolittle.Runtime.Management.GraphQL</AssemblyName>
        <RootNamespace>Dolittle.Runtime.Management.GraphQL</RootNamespace>
```